unitTests/rest/SearchEndpointTest.ts

```diff
@@ -639,6 +640,37 @@ export function SearchEndpointTest() {
           });
         });
 
+        it('for facetSearch', done => {
+          const promiseSuccess = ep.facetSearch({
+            field: 'test'
+          });
+
+          expect(jasmine.Ajax.requests.mostRecent().url).toContain(ep.getBaseUri() + '/facet?');
+          expect(jasmine.Ajax.requests.mostRecent().url).toContain('organizationId=myOrgId');
+          expect(jasmine.Ajax.requests.mostRecent().url).toContain('potatoe=mashed');
+
+          expect(jasmine.Ajax.requests.mostRecent().method).toBe('POST');
+          expect(jasmine.Ajax.requests.mostRecent().params).toContain(JSON.stringify({ field: 'test' }));
+
+          // Not real extensions, but will suffice for test purpose
+          promiseSuccess
```

Comment:
I assume this is to check if what you received from your API call is good, 
I'm wondering what you mean by your comments

> // Not real extensions, but will suffice for test purpose

and

>// Not real completions, but will suffice for test purpose

Is it because we don't really contact the API or is there another reason?

Reply:
Pretty much that, we're faking the request/response just to see if our code reacts in the correct way, this comment is there for most of the test so I just kept it here

Reply:
ok thx